client/archonMaker.js

```diff
@@ -252,16 +213,12 @@ export const buildDeckList = async (deck, language, translate, allCards) => {
                 .scaleToHeight(cardData.size);
             canvas.add(anomalyImage);
         }
-    });
-
-    return canvas.toDataURL('image/jpeg');
+        canvas.renderAll();
+    }
+    return canvas.toDataURL({ format: 'jpeg', quality: 1 });
 };
 
-/**
```

Comment:
Why you delete my jsdocs

Reply:
https://github.com/keyteki/keyteki/pull/1781/commits/27e12cb523fb0fe8eb0937c8c0007eba84924ed1